### PR TITLE
Add graphs to readme.md

### PR DESCRIPTION
**Pull request by benaadams**
Adds graphs shown on https://github.com/benaadams/d3renderer pulls data
from this repo

Needs PR #50 else images will be broken as no data in repo

~~Change version path (e.g. v1) to cache bust github's image cache - may
need to update csv files in earlier commit~~

##### Plaintext
plaintext

##### JSON
json

##### Fortunes
frotunes

##### Single Query
singlequery

##### Multi Query
multiquery

### CODE REVIEW

**Comment by benaadams** [comment]
@DamianEdwards any tweaks to graphs?

**Comment by benaadams** [comment]
If the data is in this repo you can check what the graphs look like before merging at https://github.com/benaadams/d3renderer/blob/master/README-aspnet.md

**Comment by simonmurdock** [comment]
Hello, sorry to butt in, but I thought I could provide some feedback...

Use a legend so there are fewer text overlapping issues
Display the value of the point on hover, as some are so far away from the Y-Axis It's hard to tell

**Comment by benaadams** [comment]
@simonmurdock doesn't look like can add hover/tooltips - github only allows "images" in markdown -luckily svg counts as an image. I have added `<title />` element to the lines and points; however while the work on the raw svg alas with they don't seem to work when it is used as `` or they get lost in translation in github's "user content" image cache

**Comment by simonmurdock** [comment]
I was hoping SVG would allow some fancyness like that. Shame that Github remove it. Maybe they're doing it on purpose for security etc.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,21 @@
...
+![json](https://d3renderer.azurewebsites.net/json/v1)
+
+##### Single Query
+![singlequery](https://d3renderer.azurewebsites.net/singlequery/v1)
+
+##### Multi Query
+![multiquery](https://d3renderer.azurewebsites.net/multiquery/v1)
+
+##### Fortunes
+![frotunes](https://d3renderer.azurewebsites.net/fortunes/v1)
```
Comment (Eilon) [inline README.md]: Typo in link name :smile:

**Pull request closed without merging**